src/lib/operators/join_sort_merge.cpp

```diff
@@ -62,6 +63,11 @@ std::shared_ptr<const Table> JoinSortMerge::_on_execute() {
                    !_secondary_predicates.empty(), left_input_table()->type(), right_input_table()->type()}),
          "JoinSortMerge doesn't support these parameters");
 
+  std::shared_ptr<const Table> left_input_table_ptr;
```

Comment:
Why in four lines?

Reply:
I think (correct me if we use somewhere differently) we use the ending `_ptr` only for raw pointers.

Reply:
Nope. We just generally try to avoid it because it does not really add any valuable information.